LOSTANDFOUND.md

```diff
@@ -100,3 +100,14 @@ Author(s):
 Reason for removal:
 - code broken and excluded from all tests for a long time
 - no maintainer available
+
+### gnrc_pktbuf_duplicate_upto() [b83430aa625a1d42f11f9badf5e5cfbb8efacd99]
```

Comment:
Looks like there's some rebase issue with this PR.

Reply:
it is just that the PR shows the new commits in master that are not yet in suit-dev